examples/tts/conf/talknet-pitch.yaml

```diff
@@ -11,6 +11,10 @@ model:
   dropout: 0.1
   separable: true
 
+  # F0 stats (depends on dataset, current for LJSpeech)
+  f0_mean: 178.7767791748047
```

Comment:
we can precalculate it for any dataset in data layer init, then save it as part of checkpoint (like torch bufffer)

Reply:
@stasbel it is good idea for small dataset, but for large dataset it can be impractical.

Reply:
f0 statistics calculations not just fast, it's very very fast
we cache all the f0 values now, until this is happen we can calculate all the statistics immediately